src/jennifer/adapter.cr

```diff
@@ -26,12 +26,7 @@ module Jennifer
     end
 
     def self.adapter
-      @@adapter ||= begin
-        a = adapter_class.not_nil!.build
-        self.adapter = a
-        a.prepare
-        a
-      end
+      Jennifer::Adapter::AdapterRegistry.adapter
```

Comment:
have this static call defer to the default instance in the registry. Uses of this method will need to be refactored away.